src/opendr/engine/target.py

```diff
@@ -68,6 +122,17 @@ def __init__(self, prediction: int, description=None, confidence=None):
         self.confidence = confidence
         self.description = description
 
+    @Target.data.setter
```

Comment:
Referencing the base class (`Target`) is inconsistent with how properties are being used in `data.py`.

Reply:
Fixed.